setUpApplicationDefaults.js

```diff
@@ -64,7 +69,25 @@ export default async (app) => {
     // Attach always useful middleware
     app.use(helmet());
     app.use(cors());
-    app.use(prometheus());
+
+    // Add prometheus monitoring metrics
+    if (config.prometheusMetrics) {
```

Comment:
Reasonable defaults: is it on purpose that we default to not having a prometheus metrics app at all? That may be reasonable enough, I haven't thought it through...

But I want the general approach of this lib to be "don't worry, we do the good thing per default, but you can config it or toggle it off" - I don't know if we're hitting that here with default-off.

Reply:
Agree. [5fc3eae](https://github.com/derangeddk/dexpress/pull/4/commits/5fc3eae4934ab82c51e608e42f021e368b014a15)